### PR TITLE
Automate website publication

### PR DESCRIPTION
**Pull request by mikalstill**

### CODE REVIEW

File: .github/workflows/publish-website.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  push:
+    branches:
+      - master
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          submodules: "recursive"
```
Comment (jackadamson) [inline .github/workflows/publish-website.yml:13]: Do we need submodules? But otherwise looks good

Comment (mikalstill) [reply .github/workflows/publish-website.yml:13]: We do not. I shall remove that.

**Review by jackadamson — APPROVED** [review]
Looks good to me

**Review by mandoonandy — APPROVED** [review]
Looks believable.

**Pull request merged**